index.html

```diff
@@ -5,8 +5,42 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Henry Cadena Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <h1>Henry Cadena Portfolio Mobile First</h1>
+    <header class="container-header">
+      <h3 class="logo">Henry Cadena</h3>
+      <div class="container-burger">
+        <label class="burger1"></label>
+        <label class="burger2"></label>
+        <label class="burger3"></label>
+      </div>
+    </header>
+    <main class="container-main">
+      <img class="hero-image" src="assets/background-mobile-version.svg" alt="" />
+      <div>
+        <h2>I'm Henry Cadena. Glad to see you!</h2>
```

Comment:
- [ ] For better semantic structure and improved SEO performance, it is recommended to have only one `<h1>` tag on each page. 🙏👌
- [ ] In this case, please use `<h1>` instead of <`h2>` as the page header. Having a single <`h1>` tag helps search engines understand the main topic or heading of the page, contributing to better search engine optimization. 
- [ ] By using `<h1>` as the page header, you maintain semantic consistency and enhance the overall structure of your web pages.🥱🤔🤔